Source/WebInspectorUI/UserInterface/Views/SingleSidebar.css

```diff
@@ -35,18 +35,28 @@
     top: var(--navigation-bar-height);
 }
 
-.single-sidebar.trailing > .resizer {
+.single-sidebar.trailing > .resizer.vertical-rule {
     inset-inline-start: -3px;
 }
 
-.single-sidebar.leading > .resizer {
+.single-sidebar.leading > .resizer.vertical-rule {
     inset-inline-end: -3px;
 }
 
 .single-sidebar.leading {
     border-inline-end: 1px solid var(--border-color);
 }
 
-.single-sidebar.trailing {
+body:not(.narrow) .single-sidebar.trailing {
     border-inline-start: 1px solid var(--border-color);
 }
+
+body.narrow .single-sidebar.trailing {
+    border-top: none;
+    flex-grow: 1;
+}
+
+.single-sidebar.trailing .resizer.horizontal-rule,
```

Comment:
should this be prefixed with `body:not(.narrow)`?

Reply:
No, since the horizontal rule is always hidden on `SingleSidebars` (since the parent sidebar takes care of vertical resizing).